### PR TITLE
Pass showlegend/template kwargs to plotly

### LINKED ISSUE #121: Legend labels are not visible while using interactive plots
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

Bug description

I have discrete auxiliary variables that I am using to color my embeddings. The color works fine, but I do not know which color corresponds to which label. I tried adding a legend like I would in plotly.graph_objects.Scatter, this did not produce an error, but it also does not produce a legend. How can I get a label to be associated with the embedding_label colors? Below is my code for generating the plot. 'crop_bx' is a 1xN vector.

```python
cebra_model = cebra.CEBRA(
    model_architecture = "offset10-model-mse",
    batch_size = 512,
    temperature_mode="auto",
    learning_rate = 0.0001,
    max_iterations = 1000,
    time_offsets = 10,
    output_dimension = 3,
    conditional='time_delta',
    device='cuda_if_available',
    distance='euclidean',
    verbose=True,
)

cebra_model.fit(crop_neural, crop_bx)
embedding = cebra_model.transform(crop_neural)

fig = cebra.integrations.plotly.plot_embedding_interactive(embedding, embedding_labels= crop_bx, title = "CEBRA-Behavior", cmap='tab20', legend=np.unique(crop_bx).astype(str), showlegend = True)

fig.show()
```

### Operating System

Windows 10

### CEBRA version

cebra version: '0.3.1'

### Device type

cpu

### Steps To Reproduce

_No response_

### Relevant log output

_No response_

### Anything else?

_No response_

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/daptiveMotorControlLab/CEBRA/blob/main/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by stes**
Fix #121

### CODE REVIEW

**Review by MMathisLab — APPROVED** [review]
lgtm, although I did not test it...

**Comment by nastya236** [comment]
I think there is still a problem because of how Plotly interprets the color parameter in the marker dictionary, especially when it's linked with discrete labels. I think to how discrete labels the only solution it iteratively go through the unique labels and and the trace to the scetter plot. I made a PR here (branched from @stes branch).

```
plot_embedding_interactive(embedding, labels, markersize=10, showlegend=True)
```

```
labels = array([5, 8, 5, 4, 6, 8, 2, 3, 1, 7, 2, 9, 7, 3, 5, 6, 5, 9, 8, 4, 4, 7,
       2, 0, 1, 9, 5, 7, 5, 4, 9, 5, 1, 5, 3, 6, 4, 6, 3, 6, 3, 6, 0, 0,
       4, 7, 9, 1, 1, 5, 5, 4, 0, 0, 0, 6, 8, 9, 9, 4, 4, 4, 5, 2, 4, 2,
       7, 7, 6, 3, 5, 8, 0, 3, 6, 7, 9, 9, 1, 6, 9, 7, 4, 8, 7, 6, 5, 4,
       4, 0, 2, 9, 6, 6, 9, 2, 3, 6, 7, 6])
```
Screenshot 2024-01-04 at 12 51 45

I created a [PR#123](https://github.com/AdaptiveMotorControlLab/CEBRA/pull/123) to @stes branch.

**Comment by stes** [comment]
spelling error, fixed
